docs/software/firmware/csv_format.md

```diff
@@ -0,0 +1,64 @@
+# Format specification for the internal CVS format
+
+In addition, the will be a json file per CSV file containing some metadata. 
+The file needs to be specified later - still there is some data needed here - 
+so I will list it.
+
+## Metadata
+
+OBSVersion
+: v0.3.999
+
+FormatVersion
+: 2
+
+DatasPerMeasurement
+: 3 (fix as of now is `Tms<n>`, `Lus<n>` and `Rus<n>`)
+
+MaximumMeasurementsPerLine
+: 40 (currently fix)
+
+## CSV
+
+The CSV File must contain a header line with headline entries for each field, the 
+names must be the same as given here. The number of data entries per line
+can differ.
+
+NOTE: The order of the fields is different than in Version 1 of the CVS file. Ok??
+
+Headline  | Format | Range | Sample | Description |
+---       | --- | --- | --- | --- |
+`Date`    | TT.MM.YYYY | | 24.11.2020 | UTC, typically as received by the GPS module in that second. If there is no GPS module present, system time is used. If there was no reception of a time signal yet, this might be unix time (starting 1.1.1970) which can be at least used as offset between the csv lines.    
+`Time`      | HH.MM.SS | | 12:00:00 | UTC time, see also above
+`Latitude`  | Float |  | 9.123456 | Latitude as degrees
+`LatitudeDir` | Char | `N` /  `S` | N | Latitude direction as reported by the GPS module (GPGGA)
+`Longitude` | Float |  | 42.123456 | Longitude in degrees
+`LongitudeDir` | Char | `E` / `W` | W | Longitude direction as reported by the GPS module (GPGGA)
+`Altitude`  | Float | -9999.9 - 17999.9 | 480.12 | meters above mean sea level (GPGGA)
+`Course`    | Float | 0 - 359.9 | 42 | Course over ground in degrees (GPRMC)
+`Speed`     | Float | 0 - 359.9 | 42.0 | Speed over ground in km/h
+`HDOP` | Float | 00.0 - 99.9 | 2.3  | Relative accuracy of horizontal position (GPGGA)
+`Satellites` | Integer | 0-99 | 5 | Number of satellites in use (GPGGA)
+`BatteryLevel` | Float | 0-9.99 | 3.3 | Current battery level reading (~V)
+`Left`      | Integer | 0-999 | 150 | Left minimum measured distance in centimeters of this line, 999 for no measurement. 
+`Right`     | Integer | 0-999 | 150 | Right minimum measured distance in centimeters of this line, 999 for no measurement.
+`Confirmed` | Boolean | 0-1 | 1 | Measurement was confirmed overtaking by button press  
```

Comment:
We should additionally clarify whether this measurement was corrected for the distance sensor <-> end-of-handlebar.
(I assume it is.)